lib/order.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'menu'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.